src/dash-table/derived/table/fragments.tsx

```diff
@@ -65,4 +74,4 @@ export default (
             [isEmpty(fixedColumnCells), isEmpty(cells)]
         ]
     };
-};
\ No newline at end of file
+});
```

Comment:
Fragments did *not* respect immutability of cells/filters/headers factories output. Switch out `splice` for `slice`